### ISSUE TITLE
Eyeballs hard to use when colorblind.

### ISSUE THREAD

**Issue by deviousgeek** [gitterHQ/gitter#892]
I'm (partially) color blind, making it sometimes hard to distinguish between certain colours / shades. 
As such I need to move up close to the screen and squint a little to tell the difference between the green and yellow eyeball indicators. I'm sure others have also had this issue. Something like a grey / blue color for not looking and (keep) green for looking would work, or any other two colors that stand out from each other. #color

**Comment by MadLittleMods** [comment]
Hey Tim,

Thank you for the report. There should be another difference between the two besides color to really mitigate this issue. Something like this would probably work good:

![](http://i.imgur.com/6EiB7XU.png)

**Comment by deviousgeek** [comment]
Hi Eric,

That looks like it would fit the bill perfectly, even at a great distance! :)

**Comment by MadLittleMods** [comment]
This [has been added](https://github.com/troupe/gitter-webapp/commit/3c37efe53fef1bc762fa19c5a93b14c52f107d64) and will appear when we cut the next release :)

![](http://i.imgur.com/f2yiIoY.png)
